### PR TITLE
Adding postman collections and readme

### PR DESCRIPTION
**Pull request by niick2112**

### CODE REVIEW

**Comment by Lexcess** [comment]
Had a look through, I can't see anything obviously wrong here. I know Si has had a bit more use with PostMan Collections so I asked him to take a look as well.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Postman is a GUI REST client. By importing the collections you can explore our APIs and get a better understanding of the requests and responses.
+
+## Getting Started
+
+You will need an authentication token to make any requests. For existing customers, you can get your token from our [Self Service Portal](https://portal.experianmarketingservices.com/). If you're not an existing customer, visit [edq.com](http://edq.com) for more information about our products and how to contact us.
+
+To get started with the Postman collections you will need to download the Postman tool from [getpostman.com/postman](https://www.getpostman.com/postman).
+
+With Postman installed, you can clone this repository or download the individual files. You can then import the files into Postman and start making your requests.
```
Comment (sdcondon) [inline README.md]: Check [this](https://github.com/postmanlabs/postman-app-support/issues/751) out. The implication is that these days it's possible to provide a link that'll open up Postman with particular settings. In particular, it *might* be possible to provide a URL that'll open up postman (assuming it's installed) and immediately import the collection from its raw github URL. Definitely beyond the scope of this PR, but worth investigating.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+The collections are configured to authenticate using a global **Auth-Token** variable that is sent with the request header. You will need to set this variable by clicking the eye icon in the top right corner of the Postman client. You can then add a new entry with Auth-Token as the key and your token as the value.
+
+## REST APIs
+
+Our real-time REST APIs give you flexibility and customization when verifying and enriching your contact data before it hits your database.
```
Comment (sdcondon) [inline README.md]: No links to relevant product info on edq.com? Not a big deal.

**Comment by sdcondon** [comment]
Just something to mull over - you can import Swagger docs to Postman these days (though it's not something I've ever tried) - and Swagger (sorry, Open API) is popular enough now that I don't think it's going away any time soon. Have we considered that as an approach as well as / instead of maintaining these collections?

File: address-validate-soap.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,442 @@
...
+		"_postman_id": "64db4b4d-ccdc-46ce-4642-94a9d5d5fda0",
+		"description": "Address Validate API provides a SOAP interface for global address searching and validation. We currently support over 160 countries.",
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "DoCanSearch",
+			"request": {
+				"url": {
+					"raw": "https://ws.ondemand.qas.com/ProOnDemand/V3/ProOnDemandService.asmx?WSDL",
```
Comment (sdcondon) [inline address-validate-soap.postman_collection.json]: Don't need the ?WSDL when actually invoking the service. Same for all the others.

File: address-validate-soap.postman_collection.json [hunk 4]
```diff
@@ -0,0 +1,442 @@
...
+						"ondemand",
+						"qas",
+						"com"
+					],
+					"path": [
+						"ProOnDemand",
+						"V3",
+						"ProOnDemandService.asmx"
+					],
+					"query": [
```
Comment (sdcondon) [inline address-validate-soap.postman_collection.json]: Don't need the WSDL query param when invoking the service. Same for all the others.

File: address-validate-api-rest.postman_collection.json [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+		{
+			"name": "Search-Address",
+			"request": {
+				"url": "https://api.experianmarketingservices.com/capture/v1/search-address/text?query=<Address to search for>&country=<three digit country ISO code>",
+				"method": "GET",
+				"header": [
+					{
+						"key": "Auth-Token",
+						"value": "{{Auth-Token}}",
+						"description": ""
```
Comment (sdcondon) [inline address-validate-api-rest.postman_collection.json]: Looks like we're missing some descriptions - here and in other places.

File: global-intuitive.postman_collection.json [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+									"description": "ID is returned in the payload of the Search request. The full Format URL is returned, not just the ID."
+								}
+							],
+							"variable": []
+						},
+						"method": "GET",
+						"header": [
+							{
+								"key": "Auth-Token",
+								"value": "a1b234c5-a1bc-1234-a1bc-123a4567891bc",
```
Comment (sdcondon) [inline global-intuitive.postman_collection.json]: An auth-token value - just checking this isn't a real value? Might be safer to remove this altogether? Not sure if you can put variables in examples though..

File: global-intuitive.postman_collection.json [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+									"description": "Optional parameter, default is 7."
+								}
+							],
+							"variable": []
+						},
+						"method": "GET",
+						"header": [
+							{
+								"key": "Auth-Token",
+								"value": "a1b234c5-a1bc-1234-a1bc-123a4567891bc",
```
Comment (sdcondon) [inline global-intuitive.postman_collection.json]: An auth token value - not a real value I hope?

File: global-intuitive.postman_collection.json [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+					{
+						"key": "Auth-Token",
+						"value": "{{Auth-Token}}",
+						"description": ""
+					}
+				],
+				"body": {},
+				"description": "Search for an address. Returns an object containing suggested addresses based on the included parameters."
+			},
+			"response": [
```
Comment (sdcondon) [inline global-intuitive.postman_collection.json]: Liking the presence of examples - worth considering adding examples to the other collections at some point - just don't put real auth tokens in..

File: global-intuitive.postman_collection.json (hunk 6 above)
Comment (sdcondon) [inline global-intuitive.postman_collection.json]: Have actually just read the value properly - happy that this isn't likely a real value..

File: global-intuitive.postman_collection.json (hunk 7 above)
Comment (sdcondon) [inline global-intuitive.postman_collection.json]: Have actually just read the value properly - happy that this isn't likely a real value..

**Pull request merged**